consensus/poet/core/tests/test_consensus/test_wait_certificate.py

```diff
@@ -164,49 +173,60 @@ def test_create_after_wait_timer_timed_out(self):
         with self.assertRaises(ValueError):
             WaitCertificate.create_wait_certificate(
                 poet_enclave_module=self.poet_enclave_module,
+                sealed_signup_data=signup_info.sealed_signup_data,
                 wait_timer=wt,
                 block_hash="Reader's Digest")
 
-    def test_create_with_wrong_wait_timer(self):
+    def test_create_with_wrong_signup_data(self):
```

Comment:
The thing is I took MC out of the simulator. While I think these tests are meant to be applied to both, in practice they seem to only be applied to simulator.